app/dashboard/styles.scss

```diff
@@ -183,3 +183,17 @@
     color: $color-text-placeholder-grey-dark;
     opacity: 1;
 }
+
+.iframe-container {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    margin-top: 20px;
+    margin-bottom: 30px;
+
+    .osf-video {
+        width: 560px;
```

Comment:
For mobile view, the video player will be wider than all of the other content on the page. You could set it so that for mobile (it looks like `@media (max-width: 768px)` is the threshold we are using) the width is set using `vw`.